app/models/manager_refresh/inventory/persister.rb

```diff
@@ -155,12 +155,13 @@ def add_remaining_inventory_collections(defaults, options = {})
 
   def to_raw_data
     collections_data = collections.map do |key, collection|
-      next if collection.data.blank?
+      next if collection.data.blank? && collection.manager_uuids.blank? && collection.all_manager_uuids.nil?
```

Comment:
collection.all_manager_uuids = [] will delete everything. While nil is ignored

collection.manager_uuids [] or nil is equivalent, it should be marking that the collection will not be saving any data.